frontend/src/utils/constants.ts

```diff
@@ -45,3 +45,5 @@ export const ToastTimeout = Object.freeze({
   SUCCESS: 3000,
   WARNING: 5000
 });
+
+export const MAX_TAGS = 9;
```

Comment:
Alphabetical ordering.
Also good to put in a quick JSDoc style comment on why it's 9 (max 10 tags in S3, one always used by coms, etc)